catalog.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона: Каталог (by BYN)</title>
+    <link rel="stylesheet" href="css/styles.css">
+  </head>
+
+  <body class="main-page">
+    <header> <!--Шапка-->
+      <nav> <!--Навигация по сайту-->
+      <ul> <!-- формат задать позже в CSS class="site-navigation"-->
+        <li><a href="blank.html">Информация</a></li>
+        <li><a href="blank.html">Фото и видео</a></li>
+        <li><a href="blank.html">Карта штата</a></li>
+        <li><a>Гостиницы</a></li>
+      </ul>
+      <a href="index.html"> 
+              <!--формат задать позже в CSS class="main-headder-logo"-->
+        <img src="img/city-of-sedona-logo.svg" width="170" height="70" alt="Логотип сайта">
+      </a>
+    </nav>
+
+  </header>
+
+    <main>
+      <section> <!--Условия для выбора гостиницы-->
+        <form action="https://echo.htmlacademy.ru/">
+          <div>
+            <h3>инфраструктура:</h3>
```

Comment:
Рекомендация: есть специальные теги формы - `fieldset` и `legend`. Тег `fieldset` используется для группирования схожих по предназначению полей формы, а тег `legend` используется для создания заголовка внутри `fieldset`.

Соответственно этот код можно переписать так:
```suggestion
          <fieldset>
            <legend>инфраструктура:</legend>
```